ccloud/docs/includes/replicator-connect-worker-to-destination.rst

```diff
@@ -0,0 +1,24 @@
+The destination cluster is your |ccloud| cluster, and |crep| needs to know how to connect to it.
+Use the prefix ``dest.`` to set these configuration parameters.
+
+.. sourcecode:: bash
+
+   # Confluent Replicator license topic must have replication factor set to 3 for |ccloud|
+   confluent.topic.replication.factor=3
+
+   # New user topics that |crep-full| creates must have replication factor set to 3 for |ccloud| 
+   dest.topic.replication.factor=3
+
+   # Connection information to Confluent Cloud
+   # Remaining configuration is inherited from the Connect worker's 'producer.' configuration
+   dest.kafka.bootstrap.servers=<bootstrap-servers-destination>
```

Comment:
@thomaskwscott is ` dest.kafka.bootstrap.servers` still required, as shown here?